src/test/resources/application.yml

```diff
@@ -96,7 +96,7 @@ app:
     replyTo: dev@library.tamu.edu
   # edu.tamu.weaver.reporting.controller.ReportingController
   reporting:
-    address: dev@tdl.or
+    address: dev@tdl.org
```

Comment:
Let's update this and above to `helpdesk@library.tamu.edu`. It is a overlooked copy and paste from Vireo.